### ISSUE TITLE
License discussion

### ISSUE THREAD

**Issue by doraqmon** [UBC-MDS/Rcat#70]
Hi team, we are using MIT for our package right now.
Let's discuss if there is a better license we can use.
Feel free to comment below.

**Comment by HanyingZhang** [comment]
Same as said in `pythcat` package: I think MIT will be appropriate for this package. It is simple and very permissive. As we want to keep the package open-source, MIT is a good choice with fewer restrictions. Anyone who would like to build on this package will be able to work on the code privately or publicly as long as credit is given in the license file along with a copyright notice. Besides, it is a widely-used license that contributors and users are more likely to be familiar with.

**Comment by TBarasch** [comment]
Agreed, this is an incomplete package which can be built upon, should be available both for usage and for anyone looking to build upon something.

**Comment by EitharAlfatih** [comment]
Yes. I agree with continuing with MIT now since it is the License with the lowest risk for open source packages. Maybe later when adding more functionality we might think about other options such as  Apache License 2.0

**Comment by doraqmon** [comment]
Perfect! We will keep MIT license for our package.
